src/qt/test/test_main.cpp

```diff
@@ -53,6 +53,10 @@ int main(int argc, char *argv[])
     SetupNetworking();
     SelectParams(CBaseChainParams::MAIN);
     noui_connect();
+    ClearDatadirCache();
```

Comment:
Any reason to only do this for Qt?

Reply:
We already do it on `src/test/test_bitcoin.cpp`